### PR TITLE
Card#43 fix/implement new unit tests

### PR DESCRIPTION
**Pull request by fein91**
Moved all entities string literals to class constant fields

### CODE REVIEW

**Comment by fein91** [comment]
@xelamanster, @Grechka, @amakogon check this shit!

File: src/main/java/com/smartnews/model/Article.java [hunk 1]
```diff
@@ -3,32 +3,38 @@
...
     public static final String FIND_ALL = "Article.findAll";
+    public static final String TABLE_NAME = "article";
+    public static final String SEQUENCE_GENERATOR = "article_seq_gen";
+    public static final String SEQUENCE_NAME = "article_seq";
+    public static final String FOLDER_FK = "folder_fk";
+    public static final String TAG_JOINT_TABLE_NAME = "article_tag";
+    public static final String TAG_JOINT_COLUMN_NAME = "article_fk";
+    public static final String TAG_JOINT_REFERENCED_COLUMN = "id";
+    public static final String TAG_INVERSE_JOINT_COLUMN_NAME = "tag_fk";
+    public static final String TAG_INVERSE_JOINT_REFERENCED_COLUMN = "id";
```
Comment (amakogon) [inline src/main/java/com/smartnews/model/Article.java]: maybe private?

**Comment by amakogon** [comment]
that's all from me

**Comment by amakogon** [comment]
lgtm

**Pull request merged**